script/configs/custom_analysis.yaml

```diff
@@ -0,0 +1,40 @@
+# Plugins that deliberately use their own analysis_options.yaml.
+# This only exists to allow incrementally switching to the newer, stricter
```

Comment:
nit, newline between summary and details

Reply:
Done